VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -61,21 +61,21 @@ open class VideoSettings: NSObject, NSCoding
     
     // MARK: Public API
     
-    open func parameterDictionary() -> [String: Any]
+    @objc open func parameterDictionary() -> [String: Any]
     {
         var parameters: [String: Any] = [:]
         
-        if let title = self.title, title.characters.count > 0
+        if let title = self.title, title.count > 0
```

Comment:
I'm not sure what level of changes we're comfortable making on this release branch, but we should be able to use `isEmpty` on these strings as well. Or the inverse, I suppose: `!title.isEmpty`.